src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] totals = calculateTotals(fromFileName);
```

Comment:
create constants for: comma, supply, buy, result

Your implementation should include 3 methods: read from file, calculate report, write into file
All these methods you should call in getStatistic()